extensions/CoreWidgetProvider/Widgets/SSHWalletWidget.cs

```diff
@@ -149,12 +150,26 @@ private void HandleConnect(WidgetActionInvokedArgs args)
         var cmd = new Process();
         cmd.StartInfo = new ProcessStartInfo
         {
-            FileName = "cmd.exe",
-            Arguments = $"/k \"ssh {args.Data}\"",
+            FileName = "wt.exe",
+            Arguments = $"ssh {args.Data}",

```

Comment:
```suggestion
            FileName = "ssh.exe",
            Arguments = $"{args.Data}",

```

(then you cam probably also drop the fallback below)

This will let the OS spawn `ssh` in the user's default terminal app, regardless of if it's set to Terminal or conhost.

Reply:
In the original change from cmd to wt, I got to remove the /k switch that keeps the window open. Do you know if there's something I can pass into ssh.exe to do the same thing?

Reply:
Hmm. That's a good question. The terminal has the `closeOnExit` setting, which is perfect for keeping the window open when ssh disconnects, but that isn't going to help conhost at all. 

I suppose you could _always_ just run `cmd /k ssh {{whatever}}`. Launching `cmd` directly will do the same thing - boot into whatever your default terminal is. 

In my **totally biased** opinion though, I'd just launch `ssh.exe` directly. If the users want the console window to stay open after the session disconnects, then "here's a nickel kid, go get yourself a modern Terminal (and stop using conhost)"